userservice/src/main/java/com/infodation/userservice/services/NotificationServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.infodation.userservice.services;
+
+import com.infodation.userservice.mapper.NotificationMapper;
+import com.infodation.userservice.models.User;
+import com.infodation.userservice.models.notimodel.CreateNotificationDTO;
+import com.infodation.userservice.models.notimodel.Notifications;
+import com.infodation.userservice.repositories.NotificationRepository;
+import com.infodation.userservice.repositories.UserRepository;
+import com.infodation.userservice.services.iservice.INotificationService;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class NotificationServiceImpl implements INotificationService {
+    private final NotificationRepository notiRepository;
+    private final UserRepository userRepository;
+
+    public NotificationServiceImpl(NotificationRepository notiRepository, UserRepository userRepository) {
+        this.notiRepository = notiRepository;
+        this.userRepository = userRepository;
+    }
+
+    @Override
+    public Notifications saveNoti(CreateNotificationDTO notiDTO) {
```

Comment:
I prefer the name of method to be more clear for reader
```suggestion
    public Notifications saveNotification(CreateNotificationDTO notiDTO) {
```